src/tests/nativeaot/SmokeTests/HardwareIntrinsics/Program.cs

```diff
@@ -22,7 +22,7 @@ static int Main()
 
         long lowerBound, upperBound;
         lowerBound = 1200 * 1024; // ~1.2 MB
-        upperBound = 1600 * 1024; // ~1.6 MB
+        upperBound = 1700 * 1024; // ~1.7 MB
```

Comment:
what's the reason for this change?

Reply:
this test checks for size_on_disk of the test binary. Regions adds more code which puts it over the test threshold.